catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntryQueryException.java

```diff
@@ -17,7 +17,7 @@
 /**
  * Thrown by a {@link GeoEntryQueryable} when an error occurs while querying a GeoNames resource.
  */
-public class GeoEntryQueryException extends RuntimeException {
+public class GeoEntryQueryException extends Exception {
```

Comment:
@ahoffer I was converting it to a checked exception instead of unchecked in reference to @bdeining . Is there any particular reason we would leave it as unchecked? Would it be better to leave it as unchecked?